workspaces/topology/plugins/topology/package.json

```diff
@@ -44,12 +44,14 @@
     "@backstage/plugin-kubernetes-react": "^0.4.4",
     "@backstage/plugin-permission-react": "^0.4.27",
     "@backstage/theme": "^0.6.0",
+    "@emotion/react": "^11.5.0",
+    "@emotion/styled": "^11.3.0",
```

Comment:
I see these were added but I don't see them being used in code, can we remove these?

Reply:
Removed extra unnecessary dependencies. Thanks.